src/types.ts

```diff
@@ -612,6 +625,10 @@ interface ElectronMockInstance extends Omit<Mock, Override> {
    * Current context of the mock. It stores information about all invocation calls and results.
    */
   mock: ElectronMockContext;
+  /**
+   * Used internally to distinguish the electron mock from other mocks.
```

Comment:
Optional: not sure if it is worth annotating it with private so users don't get the idea of using it. I guess you can't guarantee this from happening anyway.
```suggestion
   * Used internally to distinguish the electron mock from other mocks.
   * @private
```

Reply:
Good idea 🤝